src/main/java/junit/runner/Version.java

```diff
@@ -9,7 +9,7 @@ private Version() {
 	}
 
 	public static String id() {
-		return "4.13-SNAPSHOT";
+		return "4.14-SNAPSHOT";
```

Comment:
Do you mind moving this change it's own commit (before the other changes)?

Reply:
Sure thing! Done.